shared/model/app_stack.go

```diff
@@ -1,27 +1,71 @@
 package model
 
+import (
+	"strings"
+
+	"github.com/chanzuckerberg/happy/shared/config"
+	"github.com/chanzuckerberg/happy/shared/k8s"
+	"github.com/chanzuckerberg/happy/shared/util"
+)
+
 type AppStack struct {
 	CommonDBFields
 	AppMetadata // TODO: might want to change this to AppStackPayload but going with minimal columns for now
 }
 
+type StackMetadata struct {
+	Name               string            `json:"name,omitempty"`
```

Comment:
Is this the stack name? If so, we will have a duplicate field in `AppStackResponse` because `AppMetadata` has the stack name too, maybe we can find a way to consolidate